02-household-queries/.gitignore

```diff
@@ -3,4 +3,6 @@ __pycache__
 tanf.pdf
 guru_cards_for_nava.json
 chroma_db/
+chroma/
```

Comment:
Add comments to describe the purpose or differences between chroma_db and chroma